02_configure_host.sh

```diff
@@ -154,18 +154,52 @@ if [ "$MANAGE_INT_BRIDGE" == "y" ]; then
                grep -q BOOTPROTO /etc/sysconfig/network-scripts/ifcfg-${BAREMETAL_NETWORK_NAME} || (echo -e "\nBOOTPROTO=dhcp\n" | sudo tee -a /etc/sysconfig/network-scripts/ifcfg-${BAREMETAL_NETWORK_NAME})
            fi
        fi
-        sudo systemctl restart network
+       sudo systemctl restart network
     fi
 fi
 
 # If there were modifications to the /etc/sysconfig/network-scripts/ifcfg-*
 # files, it is required to enable the network service
-if [ "$MANAGE_INT_BRIDGE" == "y" ] || [ "$MANAGE_PRO_BRIDGE" == "y" ]; then
+if [ "$MANAGE_INT_BRIDGE" == "y" ] ; then
   sudo systemctl enable network
 fi
 
+# VLAN interface in the provisioning bridge
+if [ ! -z "${PROVISIONING_VLAN}" ] ; then
+    VLAN_INT=${PROVISIONING_NETWORK_NAME}.${PROVISIONING_VLAN}
+    sudo cp /var/lib/libvirt/dnsmasq/${BAREMETAL_NETWORK_NAME}.conf /etc/NetworkManager/dnsmasq.d/${VLAN_INT}.conf
+    sudo sed -i "s/${BAREMETAL_NETWORK_NAME}/${VLAN_INT}/" /etc/NetworkManager/dnsmasq.d/${VLAN_INT}.conf
+    sudo sed -i "/^bind-dynamic/d" /etc/NetworkManager/dnsmasq.d/${VLAN_INT}.conf
+    sudo sed -i 's#/var/lib/libvirt/dnsmasq/#/etc/NetworkManager/dnsmasq-shared.d/#' /etc/NetworkManager/dnsmasq.d/${VLAN_INT}.conf
+    sudo cp /var/lib/libvirt/dnsmasq/${BAREMETAL_NETWORK_NAME}.hostsfile /etc/NetworkManager/dnsmasq-shared.d/${VLAN_INT}.hostsfile
+    sudo cp /var/lib/libvirt/dnsmasq/${BAREMETAL_NETWORK_NAME}.addnhosts /etc/NetworkManager/dnsmasq-shared.d/${VLAN_INT}.addnhosts
+
+    # Remove the baremetal libvirt network
+    sudo virsh net-destroy ${BAREMETAL_NETWORK_NAME}
+    sudo virsh net-undefine ${BAREMETAL_NETWORK_NAME}
+
+    # Remove the baremetal bridge
+    sudo ifdown ${BAREMETAL_NETWORK_NAME} || true
+    if [ -e /etc/sysconfig/network-scripts/ifcfg-${BAREMETAL_NETWORK_NAME} ] ; then
+        sudo rm -f /etc/sysconfig/network-scripts/ifcfg-${BAREMETAL_NETWORK_NAME}
+    fi
```

Comment:
As we're removing the bridge, we probably also need to modify all of the libvirt VM definitions to move the baremetal nic to a different source bridge?

Reply:
Yes. I was thinking about that last night. Will work on it today and send updates.

Reply:
As you mentioned on slack, I think we may need to leave this bridge in place for the bootstrap VM